charts/dify/values.yaml

```diff
@@ -2857,3 +2869,17 @@ externalPgvector:
   address: "pgvector"
   port: 5432
   dbName: dify
+
+###################################
+# External Tencent
+# - these configs take effect only if both `externalWeaviate.enabled`, `externalQdrant.enabled` and `externalMilvus.enabled` and `externalPgvector.enabled` are set as `false` and `externalTencent.enabled` is `true`
+###################################
+externalTencent:
```

Comment:
We'll need a canonical name for this vector database instead of  `Tencent` which suggests the name of the enterprise itself